tests/testAddToCoadd.py

```diff
@@ -44,19 +44,16 @@
     display
 except NameError:
     display = False
-    Verbosity = 0 # increase to see trace
+    Verbosity = 0  # increase to see trace
 
 pexLog.Trace_setVerbosity("lsst.coadd.utils", Verbosity)
 
 try:
     dataDir = lsst.utils.getPackageDir('afwdata')
+    medMIPath = os.path.join(dataDir, "data", "med.fits")
```

Comment:
I prefer the code as written because the skip is caused by afwdata not being setup. I think it is clearer to test on that, so that the test message matches the thing being tested, and it is more forward looking in that we may decide to add tests that use additional data from afwData. That said, the I strongly prefer the naming in coadd.py, e.g. AfwDataDir, as and a capitalized name for the file being used.

An alternative you may prefer is for each test to generate the path to the file being tested itself, so the only global is AfwDataDir. For coadd.py I'm not very happy with that only because the same file is used in 3 places, but I guess I could make the subpath a module global and repeat the `os.path.join` in each test.